src/components/header.js

```diff
@@ -0,0 +1,19 @@
+import { AppBar } from "material-ui";
+import colors from "../styles";
+
+const Header = ({children, style}) => {
```

Comment:
see comment above, going to use children, but won't cause any issue (besides bad practice i suppose) if not removed. it just gets those from the props object if they effect, and is undefined if not.